include/seqan/align_split/align_split_interface.h

```diff
@@ -385,17 +420,13 @@ int _splitAlignmentImpl(Gaps<TContigSeqL> & gapsContigL,
 
     // TODO(holtgrew): Make selecting the left/right split position from interface possible? Maybe not necessary.
 
-    int bestScore = minValue<TScoreValue>() / 2;
-    unsigned bestPrefixLength = 0;
-    for (unsigned i = 0; i < length(scoutStateL.splitScore); ++i)
-    {
-        int s = scoutStateL.splitScore[i] + scoutStateR.splitScore[i];
-        if (s > bestScore)
-        {
-            bestScore = s;
-            bestPrefixLength = i;
-        }
-    }
+    auto itBegin = makeZipIterator(begin(scoutStateL.splitScore), begin(scoutStateR.splitScore));
+    auto res = std::max_element(itBegin, makeZipIterator(end(scoutStateL.splitScore), end(scoutStateR.splitScore)),
+                                [](auto lhs, auto rhs)
```

Comment:
Why putting it on separate line?
This way it can be optimised by moving instead of copying.